src/ai/ai.cpp

```diff
@@ -4392,7 +4475,7 @@ void assign_targets(sys::state& state, dcon::nation_id n) {
 					continue;
 				}
 
-				a_force_str += conservative_estimate_army_strength(state, ar.get_army());
+				a_force_str += estimate_army_offensive_strength(state, ar.get_army());
```

Comment:
un-cached

Reply:
were calculating how much attack force we have, which needs to account for leaders attack too